projects/go-lib/src/lib/components/go-side-nav/go-nav-item/go-nav-item.component.scss

```diff
@@ -0,0 +1,54 @@
+@import '~@tangoe/gosheets/base/variables';
+
+.go-nav-item {
+  align-items: center;
+  display: flex;
+  padding-right: .5rem;
+
+  &:hover {
+    background: $theme-dark-bg-hover;
+  }
+}
+
+.go-nav-item__link::before {
+  background: $base-primary;
+  border-radius: 0 4px 4px 0;
+  content: " ";
+  height: 100%;
+  left: 0;
+  opacity: 0;
+  position: absolute;
+  top: 0;
+  width: 4px;
+}
+
+.go-nav-item__link--active::before {
```

Comment:
Changing this to 
```
go-nav-item__link::before {
  &.go-nav-item__link--active {
    opacity: 1;
  }
}
```
actually doesn't work. The active icon to the left doesn't show up anymore

Reply:
@StevenUlmer yeah, my bad. It would need to be:
```
.go-nav-item__link--active:before {
  opacity: 1;
}
```
You should only need to style the active modifier, just make sure it is below the declaration of `go-nav-item__link` in the scss file.

Reply:
```css
.go-nav-item__link--active::before {
  opacity: 1;
}
```
`:` for pseudo classes. `::` for pseudo elements